test/integration/case.test.js

```diff
@@ -144,19 +144,41 @@ describe('Case', () => {
     });
   });
 
-  it('add user consent to publish', async () => {
-    // const newParams = {
-    //   caseId: caseToTestStaging.id,
-    // };
-
-    // const results = await chai
-    //   .request(server.app)
-    //   .post(`/case/consent-to-publishing`)
-    //   .set('Authorization', `Bearer ${token}`)
-    //   .set('content-type', 'application/json')
-    //   .send(newParams);
-
-    // results.should.have.status(200);
+  describe('consent to publishing case', () => {
+
+    before(async () => {
+      await casesService.deleteAllRows()
+
+      const caseParams = {
+        organization_id: currentOrg.id,
+        state: 'unpublished'
+      };
+      currentCase = await mockData.mockCase(caseParams)
+    });
+
+    it('returns the updated case', async () => {
+      const requestParams = {
+          caseId: currentCase.caseId
+      };
+
+      const result = await chai
+        .request(server.app)
+        .post(`/case/consent-to-publishing`)
+        .set('Authorization', `Bearer ${token}`)
+        .set('content-type', 'application/json')
+        .send(requestParams);
+
+        result.error.should.be.false;
+        result.should.have.status(200);
+        result.body.should.be.a('object');
+        result.body.should.have.property('case');
+        result.body.case.should.be.a('object');
+        result.body.case.should.have.property('caseId');
+        result.body.case.should.have.property('state');
+        result.body.case.should.have.property('updatedAt');
+        result.body.case.should.have.property('expiresAt');
```

Comment:
Add a check that `consentedToPublishingAt` exists on the returned case object

Reply:
The frontend has no use for the this piece of data at the moment. Its just something we store on the backend for record keeping. For that reason I feel that we should leave it out of the response payloads.

Reply:
Should `stagedAt` have similar treatment? Or is this use by the frontend?

Reply:
The frontend will be using the `stagedAt` field.

Reply:
👍